lib/agenda/index.ts

```diff
@@ -99,21 +116,21 @@ class Agenda extends EventEmitter {
   start!: typeof start;
   stop!: typeof stop;
 
-  constructor(config: any = {}, cb?: (error: Error, collection: Collection<any> | null) => void) {
+  constructor(config: AgendaConfig = {}, cb?: (error: Error, collection: Collection<any> | null) => void) {
     super();
 
     this._name = config.name;
     this._processEvery = (humanInterval(config.processEvery) ?? humanInterval('5 seconds')) as number; // eslint-disable-line @typescript-eslint/non-nullable-type-assertion-style
-    this._defaultConcurrency = config.defaultConcurrency || 5;
-    this._maxConcurrency = config.maxConcurrency || 20;
-    this._defaultLockLimit = config.defaultLockLimit || 0;
-    this._lockLimit = config.lockLimit || 0;
+    this._defaultConcurrency = config.defaultConcurrency ?? 5;
```

Comment:
again here you change the behaviour slightly by using ?? instead of || ...intended? does it make sense to set a default concurrency of 0?

Reply:
I'd second that.